.github/config.yml

```diff
@@ -25,4 +25,4 @@ newPRWelcomeComment: >
   Of course, if you want to opt-out this time there is no
   problem at all with adding your name later.
   You will be always welcome to add it in the future whenever
-  you feel it should be listed.
\ No newline at end of file
+  you feel it should be listed.
```

Comment:
Please leave out for another PR. Happy to review a housekeeping PR with these kinds of stylistic changes only.